modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/SelectBenchmark.java

```diff
@@ -172,7 +204,40 @@ private org.apache.ignite.sql.ResultSet<SqlRow> sql(String sql, Object... args)
     }
 
     /**
-     * Benchmark state for {@link #sqlThinGet(SqlThinState)}.
+     * Benchmark state for {@link #sqlGet(SqlState, Blackhole)}.
+     *
+     * <p>Holds {@link Session}.
+     */
+    @State(Scope.Benchmark)
+    public static class SqlScriptState {
```

Comment:
to make fair comparison, we need one more state: the one that uses internal API as well, but instead executes statements with `querySingleAsync`

Reply:
Thanks. Added a separate `sqlGetInternal` benchmark that uses the internal API.